kibana/config/entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 # Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
 
 set -e
-
+export ELASTICSEARCH_HOSTS="[$ELASTICSEARCH_HOSTS]"
```

Comment:
convert ELASTICSEARCH_HOSTS string to array to easy input for ELASTICSEARCH_HOSTS env